app/data/schema/schema-v1.json

```diff
@@ -1,258 +1,265 @@
 {
```

Comment:
The reformatting it should be done in a separate commit, it's not clear what changes (if any) there are.